guides/v3.5.0/models/index.md

```diff
@@ -54,9 +54,25 @@ more information about how components get model data, see the
 [Specifying a Route's Model](../routing/specifying-a-routes-model/)
 guide.
 
+At first, using Ember Data may feel different than the way you're used
+to writing JavaScript applications. Many developers are familiar with
+using AJAX to fetch raw JSON data from an endpoint, which may appear
+easy at first. Over time, however, complexity leaks out into your
+application code, making it hard to maintain.
+
+With Ember Data, managing models as your application grows becomes both
+simple _and_ easy.
+
+Once you have an understanding of Ember Data, you will have a much
+better way to manage the complexity of data loading in your application.
+This will allow your code to evolve without becoming a mess.
```

Comment:
```suggestion
This will allow your code to evolve and grow, with better maintainability.
```

Agreed, if we can not use the "mess" language, it's better. Some of my past writing drew criticism for a similar statement and I think people had good points.